VenafiTppPS/Code/Public/Get-TppCertificate.ps1

```diff
@@ -121,19 +151,44 @@ function Get-TppCertificate {
 
         $params.Body.CertificateDN = $Path
 
-        if ( $PSBoundParameters.ContainsKey('IncludePrivateKey') ) {
+        if ($IncludePrivateKey.IsPresent) {
```

Comment:
please remove `.IsPresent`

Reply:
Hi @gdbarron ,
     did you mean replacing this line

```powershell
if ($IncludePrivateKey.IsPresent) {
```
with

```powershell
if ( $PSBoundParameters.ContainsKey('IncludePrivateKey')) {
'''